### PR TITLE
Rename ADO displayNames for loadtesting

### PR DESCRIPTION
**Pull request by heoelri**
This PR renames some of the displayNames used for the embedded and standalone loadtest.

### CODE REVIEW

**Comment by heoelri** [comment]
Running e2e now and will paste some screenshots here as soon as it's done. Will use the same displayNames in the other LT implementation as well to reduce the need to update screenshots later. @hjscherer FYI

File: .ado/pipelines/templates/stages-locust.yaml [hunk 1]
```diff
@@ -22,10 +22,10 @@ parameters:
 stages:
```
Comment (sebader) [inline .ado/pipelines/templates/stages-locust.yaml:22]: I guess we leave the file name untouched for now and later might have two different stage files for ALT and Locust?

Comment (heoelri) [reply .ado/pipelines/templates/stages-locust.yaml:22]: Yes. The other one already exists. For now its just about the displayNames.

**Review by sebader — APPROVED** [review]

**Comment by heoelri** [comment]
@hjscherer renamed:

image

**Pull request merged**